### DISCUSSION TITLE
Add Ubo Detector

### DISCUSSION THREAD

**Discussion by JavierFernandezOrtega** [neurodesk/neurodesk.github.io#99] [General]
**Ubo Detector**
Could this software be added? 
Here the link: https://github.com/cheba-nil/CNS

System Requirements
UBO Detector has been tested on both CentOS release 6.8 and macOS Sierra with the following software/toolboxes installed:
SPM12
FSL v5.0
MATLAB R2016a or above with the following toolboxes installed:
Image Processing Toolbox (ver 9.5 tested)
Statistics and Machine Learning Toolbox (ver 11.0 tested)
Parallel Computing Toolbox (ver 6.9 tested)

**Comment by stebo85** [comment]
Dear @JavierFernandezOrtega,

This software could be possible to be added, but we will need your help to achieve this. The software is very old with outdated dependencies and seems unmaintained, so there could be various challenges in getting it to work. The dependency on matlab is problematic, especially such an old version, because Matlab doesn't distribute these old versions via software containers - so it would need to be compiled and brought into the matlab runtime environment. 

- would you be happy to help to get it to work?
- do you have the software currently running and a test dataset where we could see if it works in any way?

Kind regards
Steffen

**Reply by JavierFernandezOrtega** [reply]
Hello @stebo85,

I have searched and found an update of the software, which might make it easier to get it to work. You can find the update at the following link: https://github.com/JiyangJiang/CNS

I would be happy to assist you in getting it to work, but currently, I do not have the software running. Nevertheless, I have been in contact with the software's support team to address the issues.

Best regards
Javier

**Reply by stebo85** [reply]
Dear @JavierFernandezOrtega - sounds great! If you can ask the developers to directly add their software to Neurodesk, that would be ideal.
